model/operation.cc

```diff
@@ -1120,6 +1146,35 @@ namespace minsky
     cairo_move_to(cairo,-9,3);
     cairo_show_text(cairo,"frac");
   }
+  template <> void Operation<OperationType::percent>::iconDraw(cairo_t* cairo) const
+  {
+    cairo_move_to(cairo,-6,3);
+    cairo_show_text(cairo,"%");
+  }
+  template <> void Operation<OperationType::gamma>::iconDraw(cairo_t* cairo) const
+  {
+    cairo_move_to(cairo,-6,3);
+    cairo_show_text(cairo,"Γ");
+  }      
+  template <> void Operation<OperationType::digamma>::iconDraw(cairo_t* cairo) const
+  {
+    cairo_move_to(cairo,-6,3);
+    cairo_show_text(cairo,"ψ");
+  }
+  template <> void Operation<OperationType::polygamma>::iconDraw(cairo_t* cairo) const
+  {
+    cairo_move_to(cairo,-7,3);
+    cairo_show_text(cairo,"ψ");
+    cairo_rel_move_to(cairo,0,-3);
+    cairo_set_font_size(cairo,7);
+    cairo_show_text(cairo,"(n)");
```

Comment:
If polygamma used the arg attribute for it's order, rather than being a binary op, you could display the actual order here.

Reply:
I sort of do that in equationDisplay.cc, have not considered it here. I guess 
I have not seen an example in which the arg attribute is used for another operation. Is it similar to how it's done in equationDisplay.cc?

Reply:
Arg was originally introduced to specify the delta in the difference operation, and the window size in the running sum.
Given  you've decided to stick with the binary function, it might still be nice to dynamically display the order in the symbol here...